src/Config/InfectionConfig.php

```diff
@@ -251,6 +258,17 @@ function ($excludeDir) use ($srcDir) {
 
     public function getTmpDir(): string
     {
-        return $this->config->tmpDir ?? sys_get_temp_dir();
+
+        if (empty($this->config->tmpDir)) {
```

Comment:
maybe we need to throw an exception for case when `"tmpDir": ""` instead of use `sys_temp_dir`? what do you think?

Reply:
it's better to continue working if we can rather than throw an exception.

I mean even if we throw an exception here, Infection should fall back to sys_temp_dir since this is not critical mistake and does not block mutation testing. WDYT?

Reply:
imho to be user friendly we need to show warning message in this case if we don't want to throw an exception. (it's not easy to do right now)
so feel free to leave it as is

Reply:
forgot about it, will do, thank you